### DISCUSSION TITLE
add own wallpapers

### DISCUSSION THREAD

**Discussion by marcdiversion** [tanujnotes/Olauncher#462] [Q&A]
Hi, I love your launcher!!! Thanks a lot.
Quick question, I don't understand how to use my own wallpapers with the app. Any tips?

**Comment by tanujnotes** [comment]
Hi, you can set your own wallpaper from your phone settings or Gallery/Photos app. You might have to google the specific steps based on your phone model (Pixel, Samsung, etc).
